tests/modal/test_models.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase
+
+from djangocms_frontend.contrib.modal.forms import ModalForm, ModalTriggerForm, ModalContainerForm, ModalInnerForm
+from djangocms_frontend.contrib.modal.models import Modal, ModalTrigger, ModalContainer, ModalInner
+
+
+class ModalModelTestCase(TestCase):
+   def test_Modal_instance(self):
```

Comment:
```suggestion
    def test_Modal_instance(self):
```
There's an indentation error here.